ingress/controllers/gce/main.go

```diff
@@ -83,6 +75,13 @@ var (
 		`Optional, if this controller is running in a kubernetes cluster, use the
 		 pod secrets for creating a Kubernetes client.`)
 
+	// TODO: Consolidate this flag and running-in-cluster. People already use
+	// the first one to mean "running in dev", unfortunately.
+	useRealCloud = flags.Bool("use-real-cloud", false,
```

Comment:
I do not get the justification for adding this flag. This is for dev purpose right?  e.g. not running in cluster but use real cloud?